src/components/Footer.js

```diff
@@ -1,7 +1,40 @@
-import React from "react";
+import logo from "@/public/assets/FooterLogo.svg";
+import Image from "next/image";
+
+import {
+  FaInstagram,
+  FaDiscord,
+  FaYoutube,
+  FaSlack,
+  FaEnvelope,
+  FaMediumM,
+} from "react-icons/fa";
 
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="bg-black flex w-full pl-6 pb-14 pt-28">
+      <div className="grid grid-cols-11 text-white justify-between w-full ">
```

Comment:
instead of 11 columns try use changing the width for each column to have it span more.